server/routes/index.js

```diff
@@ -862,107 +862,99 @@ router
       return res.status(502)
     }
   })
-  .post(ensureAdmin, function (req, res) {
-    if (Object.prototype.hasOwnProperty.call(req.body, 'role')) {
-      const { appDb } = req.app.locals
-      appDb
-        .collection('users')
-        .findOneAndUpdate(
-          { uid: req.params.uid },
-          { $set: { role: req.body.role } },
-          { returnOriginal: false },
-          function (err) {
-            if (err) {
-              console.log(err)
-              return res.status(502).send({ message: 'fail' })
-            } else {
-              return res.status(201).send({ message: 'success' })
-            }
-          }
-        )
-    } else {
+  .post(ensureAdmin, async (req, res) => {
+    try {
+      const { prisma } = req.app.locals
+      const updateUserRole = await prisma.users.update({
```

Comment:
is this an `updatedUser`? Or is this a boolean that tells us whether this worked or not?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237/commits/6580f4c44fa739706b7214079ccef4af7e744830